### PR TITLE
E2E test improvements

### PR DESCRIPTION
**Pull request by roobre**
The main objective of this PR is:
 - bumping KSM versions
 - updating Json schemas

### CODE REVIEW

File: e2e/cmd/e2e.go [hunk 1]
```diff
@@ -397,10 +396,11 @@ func executeTests(
 					}
 					return err
 				}
+				logger.Debugf("The test 'checking if specific entities match our JSON schemas' succeeded")
 				return nil
 			},
 			retry.OnRetry(func(err error) {
-				logger.Debugf("Retrying due to: %s", err)
+				logger.Debugf("Retrying since the error might be cause by the environment not being ready yet")
```
Comment (roobre) [inline e2e/cmd/e2e.go]: Can we add some identification of which scenario are we testing to this logs?

File: e2e/schema/cluster.json [hunk 2]
```diff
@@ -1,32 +1,50 @@
...
+  ],
   "required": [
-    "clusterName",
     "clusterK8sVersion",
+    "clusterName",
     "entityName",
     "event_type"
   ]
-}
+}
```
Comment (roobre) [inline e2e/schema/cluster.json:50]: EOF newline

**Review by paologallinaharbur — APPROVED** [review]

File: e2e/schema/service.json [hunk 3]
```diff
@@ -94,7 +93,6 @@
     }
   ],
   "required": [
-    "clusterIP",
```
Comment (roobre) [inline e2e/schema/service.json]: I agree with this change, but I don't understand why some services (or versions) are not returning this. Do you know the root cause?

Comment (paologallinaharbur) [reply e2e/schema/service.json]: I am investigating that

Comment (paologallinaharbur) [reply e2e/schema/service.json]: https://github.com/newrelic/nri-kubernetes/blob/b6cec8981d79fc6b49861a1358301e88d333a170/e2e/charts/test-statefulset/templates/statefulset.yaml#L8

For some reason the service created forces the clusterIP to be none

Comment (paologallinaharbur) [reply e2e/schema/service.json]: I am currently testing locally to re-add clusterIP

**Comment by paologallinaharbur** [comment]
I removed the e2e support for helm3 in favour of https://github.com/newrelic/nri-kubernetes/pull/110

**Review by roobre — COMMENTED** [review]
Looking HELLA RAD! 👍🏻

**Pull request merged**